src/coreclr/tools/aot/ILCompiler.RyuJit/JitInterface/CorInfoImpl.RyuJit.cs

```diff
@@ -2303,6 +2303,35 @@ private bool getReadonlyStaticFieldValue(CORINFO_FIELD_STRUCT_* fieldHandle, byt
             return false;
         }
 
+#pragma warning disable CA1822 // Mark members as static
```

Comment:
This warning disable should not be needed anymore now that the method is actually implemented.